app.py

```diff
@@ -242,6 +287,26 @@ def __add_write_node_commands(self, context=None):
                     context=context,
                 )
             )
-            
-            
 
+        # Show the convert actions in the Menu if configured to do so
+        if self.get_setting("show_convert_actions"):
+
+            convert_to_write_nodes_action = lambda :self.convert_to_write_nodes(show_warning=True)
+            convert_from_write_nodes_action = lambda: self.convert_from_write_nodes(show_warning=True)
+
+            self.engine.register_command(
+                "Convert All SG Write Nodes to Write Nodes...",
+                convert_to_write_nodes_action,
+                {
+                    "type": "context_menu",
+                    "icon": os.path.join(self.disk_location, "icon_256.png"),
+                }
+            )
+            self.engine.register_command(
+                "Convert All Write Nodes to SG Write Nodes...",
```

Comment:
nitpick: Given the dialogs, i think we can now say "Convert Write Nodes back to SG Format..."